datafusion/core/src/physical_plan/file_format/csv.rs

```diff
@@ -21,7 +21,6 @@ use crate::datasource::file_format::file_type::FileCompressionType;
 use crate::error::{DataFusionError, Result};
 use crate::execution::context::{SessionState, TaskContext};
 use crate::physical_plan::expressions::PhysicalSortExpr;
-use crate::physical_plan::file_format::delimited_stream::newline_delimited_stream;
```

Comment:
Can the corresponding `newline_delimited_stream` module be deleted too? 

https://github.com/search?q=repo%3Aapache%2Farrow-datafusion%20newline_delimited_stream&type=code

Reply:
Unfortunately it is still used by the schema inference logic, I'll see about resurrecting the PR to move to using the upstream implementation